docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -162,6 +164,63 @@ fwupdmgr refresh
 fwupdmgr update
 ```
 
+## Dasharo EC Transition
+
+DTS allows to perform full EC transition from vendor to Dasharo/ec FW.
+
+To perform EC transition make sure you are running DTS version 1.0.1 or higher
+and follow this steps:
+
+* After boot, choose option number 9 to drop to Shell
+* Download BIOS and EC update files. You can download them from
+[there (BIOS)](https://cloud.3mdeb.com/index.php/f/491273) and
+[there (EC)](https://cloud.3mdeb.com/index.php/f/491331)
+> Note: At the time writing this instruction, current highest version is 1.3.0
```

Comment:
If the above-mentioned problems will be improved, there's no reason to add that information.

Reply:
Links for download added https://github.com/Dasharo/docs/pull/291/commits/876bbe8f5cb9fd74e00055a7536ffeee1ea111b0#diff-b5545f47decb8260688b188363790fc5aef4ea727b9c363329f070d1fa3664a4R176